### ISSUE TITLE
Navigation component does not render

### ISSUE THREAD

**Issue by ArenaGrenade** [sametweb/react-step-builder#24]
Hi! For some reason I am unable to use a navigation component and get it to render. My setup is as follows. I am using `reactstrap` for styling.

```javascript
import { Container, Form, FormGroup, Input, Label, FormFeedback, Alert, Button } from "reactstrap";
import { Steps, Step } from 'react-step-builder'

const Navigation = (props) => {
    console.log(props.size)
    return (
        <FormGroup>
            <Button onClick={props.prev}>Previous</Button>
            <Button onClick={props.next}>Next</Button>
        </FormGroup>
    );
}

const config = {
	navigation: {
		component: Navigation, // a React component with special props provided automatically
		location: "after" // or after
	}
}

const Step1 = props => {
    return (
        <FormGroup>
            <Label for="step1">Step 1</Label>
            <Input type="text" name="step1" id="step1"></Input>
        </FormGroup>
    )
}

const Step2 = props => {
    return (
        <FormGroup>
            <Label for="step2">Step 2</Label>
            <Input type="text" name="step2" id="step2"></Input>
        </FormGroup>
    )
}

const UserForm = props => {
    return (
        <Container>
            <h1>User Registration form</h1>
            { alert.shown ? <Alert color="danger">{ alert.message }</Alert> : null }
                <Steps config={config}>
                    <Step title="Step 1" component={Step1}></Step>
                    <Step title="Step 2" component={Step2}></Step>
                </Steps>
        </Container>
    )
}

export default UserForm;
```

**Comment by sametweb** [comment]
Hello, thanks for reporting this.

Only thing I can think of is that maybe your version is not 2.0.5. Can you check your package.json file under `dependencies`?

**Comment by ArenaGrenade** [comment]
Hi! Looks like I am using `"react-step-builder": "^2.0.5"`. That should be the latest version right? I have seen your sandbox, but that had buttons on ever Step component. Does the config object work for you?

**Comment by sametweb** [comment]
It looks like my last publish had some problems compiling from TS before the npm publish. Now I published a new version 2.0.6 and tried it on the codesandbox repo and it is working. Let me know if you have any more questions.

**Comment by ArenaGrenade** [comment]
Thank you! Its a great library you have built.
